src/services/wallet-client/WalletClient.ts

```diff
@@ -16,44 +16,46 @@ class WalletClient {
   init() {}
 
   async getCurrentBalance(): Promise<BNWei> {
-    // TODO: replace placeholder
-    return new Promise(resolve => {
-      setTimeout(() => {
-        resolve(audioToWei('100000' as StringAudio))
-      }, 1500)
-    })
+    try {
+      const balance = await AudiusBackend.getBalance()
+      return balance as BNWei
+    } catch (err) {
+      console.log(err)
+      return new BN('0') as BNWei
```

Comment:
Nit: would be v nice to have a constant ZeroBNWei that we can just import from the wallet client or something